mistletoe/core_tokens.py

```diff
@@ -1,18 +1,17 @@
 import re
-
+import unicodedata
 
 whitespace = {' ', '\t', '\n', '\x0b', '\x0c', '\r'}
```

Comment:
I would like to arise a question, when do use `whitespace` set instead of `unicode_whitespace` set? Is it possible to use `unicode_whitespace` to replace this set?

I don't get too much into the code, hope this question won't sound stupid xD.